airbyte-cdk/java/airbyte-cdk/core/src/main/resources/AirbyteLogMessageTemplate.json

```diff
@@ -7,7 +7,7 @@
     },
     "message": {
       "$resolver": "pattern",
-      "pattern": "%level %C{1.}(%M):%L %m",
+      "pattern": "%level %thread %C{1.}(%M):%L %m",
```

Comment:
This being a user-visible change, it feels like it deserves an entry in the CDK changelog and therefore its own CDK version.